### PR TITLE
New package: python-netaddr-0.7.18

### PR DESCRIPTION
**Pull request by felix**

### CODE REVIEW

**Comment by pullmoll** [comment]
Change to `license="3-clause-BSD"` and add a `post_install()` with `vlicense LICENSE`.

File: srcpkgs/python-netaddr/template [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+license="3-clause-BSD"
+distfiles="${PYPI_SITE}/n/netaddr/netaddr-${version}.tar.gz"
+checksum=a1f5c9fcf75ac2579b9995c843dade33009543c04f218ff7c007b3c81695bd19
+
+post_install() {
+	vlicense LICENSE
+}
+
+pkg_install() {
+	vlicense LICENSE
```
Comment (pullmoll) [inline srcpkgs/python-netaddr/template]: No, this is the wrong place. Put the `vlicense LICENSE` inside the `pkg_install()` in `python3.4-netaddr_package()`.

Comment (felix) [inline srcpkgs/python-netaddr/template]: Done

**Comment by pullmoll** [comment]
Oops.. netaddr3.4 was not moved in the python3.4 subpkg.

**Pull request merged**